chapters/numbers-to-indices/discrete-random-vars-binomial-distribution/reading/README.md

```diff
@@ -1 +1,548 @@
+There are exactly $\binom{n}{x}$ such sequences, so the total probability of landing $x$ heads in $n$ tosses is
+
+$$\binom{n}{x}p^x(1-p)^{n-x}$$
+
+:::
+
+:::info Example
+
+Let the probability that a certain football club wins a match be equal to `0.4`.
+If the total number of matches played in the season is `30`, what is the probability that the football club wins the match `10%` of the time?
+We first calculate the number of times a match was played and won by multiplying the percentage of wins by the number of matches played:
+
+`10%` of `30` times = `3` times
+
+We can now proceed to calculate the probability that they will win the match given that their probability of a winning is `0.4`, if they play `3` times in a season.
+This can be computed as follows:
+
+$$\binom{30}{3} \cdot (0.4)^3 \cdot (1-0.4)^{30-3}$$
+
+$$=0.000265$$
+
+This can be calculated in R using the code below:
+
+```R
+> dbinom(3,30,0.4)
+[1] 0.0002659437
+```
+
+This is equal to the manual calculation using the binomial theorem.
+
+:::
+
+:::info Example
+
+Suppose a youngster puts his shirt on by himself every day for five days.
+The probability that he puts it on the right way each time is $p=0.2$.
+We let $X$ be a random variable that describes the number of times the youngster puts his shirt on the right way.
+The youngster can either put the shirt on the wrong or the right way so $X$ follows the binomial distribution with the parameters $p=0.2$ (the probability of a successful trial) and $n=5$ (number of trials).
+We can now calculate for example the probability that the youngster will put it on the right way for at least four days
+
+Putting the shirt on the right way for at least four days, means that the youngster will either put it on the right way for either four or five days (at least four or more days of five days total).
+We thus have to calculate the probability that the youngster will put his shirt on the right way for four and five days separately and then we add it together.
+We can write this process as follows:
+
+$$\begin{aligned} P(X\geq4) &= P(X=4) + P(X=5)\\ &= \binom{5}{4}\cdot0.2^4\cdot(1-0.2)^{5-4} + \binom{5}{5}\cdot0.2^5\cdot(1-0.2)^{5-5}\\ &= 5\cdot0.2^4\cdot0.8^1 + 1\cdot0.2^5\cdot0.8^0\\ &= 5\cdot0.2^4\cdot0.8 + 0.2^5\cdot1\\ & =5\cdot0.8 \cdot0.2^4 + 0.2^5\\ &= 4\cdot0.2^4 + 0.2^5\\ &= 4\cdot0.0016 + 0.00032\\ &= 0.00672\end{aligned}$$
+
+The probability that the youngster will put his shirt on the right way for at least four out of five is thus approximately `0.7%`.
+
+This is possible to calculate in R in a several ways, either using the command $\verb|dbinom|$ or $\verb|dbinom|$.
+
+The command $\verb|dbinom|$ calculates
+
+$$P(X = k)$$
+
+and the command $\verb|pbinom|$ calculates
+
+$$P(X \leq k)$$
+
+where $k$ is the number of successful trials.
+
+If $n$ is the number of trials and $p$ is the probability of a successful trials then the commands are used by writing
+
+$$\verb|dbinom(k,n,p)|$ and $\verb|pbinom(k,n,p)|$$
```

Comment:
This displays a rendering error:
![image](https://github.com/open-education-hub/ccas/assets/19818392/80fa1aae-eff4-4e15-9272-90f0a9456960)